spec/support/feature_test_helpers.rb

```diff
@@ -19,7 +19,7 @@ def strip_html_tags(text)
   end
 
   def current_tax_year
-    Rails.application.config.current_tax_year.to_i
+    TaxReturn.current_tax_year.to_i
```

Comment:
You don't need `.to_i` here because `TaxReturn.current_tax_year` calls `to_i` already.